vaadin-context-menu-flow-parent/vaadin-context-menu-flow/src/main/resources/META-INF/resources/frontend/contextMenuConnector.js

```diff
@@ -1,127 +1,107 @@
-import * as Gestures from "@vaadin/component-base/src/gestures.js";
 (function() {
-  const tryCatchWrapper = function(callback) {
+  function tryCatchWrapper(callback) {
     return window.Vaadin.Flow.tryCatchWrapper(callback, 'Vaadin Context Menu');
-  };
-
-  window.Vaadin.Flow.contextMenuConnector = {
```

Comment:
**note:** The previous `contextMenuConnector` was renamed and moved to `contextMenuTargetConnector.js`.